SensorableServicesPlatform/package.json

```diff
@@ -13,11 +13,13 @@
     "dev": "ts-node ./src/index.ts"
   },
   "dependencies": {
+    "@types/mysql": "^2.15.21",
     "@types/node": "^17.0.21",
     "debug": "^4.3.4",
     "mqtt": "^4.3.7",
     "mysql": "^2.18.1",
     "pure-uuid": "^1.6.2",
+    "tsc": "^2.0.4",
```

Comment:
What is this? We haven't used mysql and tsc until now? it's because the server, yes?

Reply:
This is because it wasn't yet installed in our server, yes!